Components/ListPosts/howto.md

```diff
@@ -243,23 +240,12 @@ add_filter('Flynt/addComponentData?name=ListPosts', function ($data, $parentData
 function addDataToAllPosts($posts) {
     if (is_array($posts)) {
         $posts = array_map(function ($post) {
-            $post = addExcerpt($post);
+            $post->excerpt = $post->get_preview(50, false, 'More');
```

Comment:
Instead of hardcoding 'More', this should show the usage of the already existing `$data['readMoreLabel']` variable.